com.github.sdbg.debug.ui/src/com/github/sdbg/debug/DebugLaunch.java

```diff
@@ -0,0 +1,37 @@
+package com.github.sdbg.debug;
+
+import com.github.sdbg.debug.ui.internal.DartUtil;
+import com.github.sdbg.debug.ui.internal.chrome.ChromeLaunchShortcut;
+import com.google.gdt.eclipse.core.sdbg.IDebugLaunch;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+
+public class DebugLaunch implements IDebugLaunch {
+
+  @Override
+  public boolean hasChromeLauncher() {
```

Comment:
I think GWT-Ecluipse-Plugin should not know that IDebugLaunch will launch the Chrome browser. Tomorrow, that could be Chrome Mobile, or Safari, or IE.

Therefore, let's change the name of this method to "hasLauncher" or maybe even "isLaunchable".
What is the purpose of this method BTW?